tests/unit/test-cmd/test.build.js

```diff
@@ -376,8 +373,12 @@ describe('build', () => {
 
     it('should reject on Unexpected errors', () => withTempDir(
       (tmpDir) => {
-        const fakeEventToPromise = sinon.spy(
-          () => Promise.reject(new Error('Unexpected error')));
+        const fakeEventToPromise = sinon.spy(async (stream) => {
+          await defaultEventToPromise(stream, 'close');
+          await fs.unlink(
+            path.join(tmpDir.path(), 'minimal_extension-1.0.zip'));
```

Comment:
This test fails because the created zip file is not named 'minimal_extension-1.0.zip'.

Let's delete the file by discovering its name inside this async function, so that if the file name changes
the test is still able to remove the tmpDir path correctly:

```diff
diff --git a/tests/unit/test-cmd/test.build.js b/tests/unit/test-cmd/test.build.js
index ca6e0e8..b483057 100644
--- a/tests/unit/test-cmd/test.build.js
+++ b/tests/unit/test-cmd/test.build.js
@@ -375,8 +375,10 @@ describe('build', () => {
       (tmpDir) => {
         const fakeEventToPromise = sinon.spy(async (stream) => {
           await defaultEventToPromise(stream, 'close');
-          await fs.unlink(
-            path.join(tmpDir.path(), 'minimal_extension-1.0.zip'));
+          const files = await fs.readdir(tmpDir.path());
+          for (const file of files) {
+            await fs.unlink(path.join(tmpDir.path(), file));
+          }
           return Promise.reject(new Error('Unexpected error'));
         });
         const sourceDir = fixturePath('minimal-web-ext');
```